### ISSUE TITLE
[semantic-issue] Add semantics for sklearn RepeatedStratifiedKFold

### ISSUE THREAD

**Issue by osryde** [spangea/Pyra#178]
**Description**
Invokes the constructor to instantiate a RepeatedStratifiedKFold object.
Notebook: 0-91-accuracy-score-for-4-models-lr-rc-gbc-rfc.py

**Minimal running example**
```
from sklearn.model_selection import RepeatedStratifiedKFold

cv = RepeatedStratifiedKFold(n_splits=10, n_repeats=3, random_state=1)
```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.RepeatedStratifiedKFold.html

**Note (optional)**
Returns an object of RepeatedStratifiedKFold type.
